### PR TITLE
added speed option on interface, e.g.: speed: 1000, speed: 10000

### PR DESCRIPTION
**Pull request by cpu-lz**
##### SUMMARY
added speed option on interface, e.g.: speed: 1000, speed: 10000

##### COMPONENT NAME
os10_interface

### CODE REVIEW

File: roles/os10_interface/templates/os10_interface.j2 [hunk 1]
```diff
@@ -114,9 +115,15 @@ interface {{ interface_key }}
  no mtu
     {% endif %}
   {% endif %}
- 
+
+  {% if intf_vars.speed is defined %}
+    {% if intf_vars.speed %}
+ speed {{ intf_vars.speed }}
```
Comment (gunasekaran-t) [inline roles/os10_interface/templates/os10_interface.j2:121]: Please add 'no' form to the else case

File: roles/os10_interface/templates/os10_interface.j2 [hunk 2]
```diff
@@ -137,7 +144,7 @@ interface {{ interface_key }}
       {% endif %}
     {% endif %}
   {% endif %}
- 
+
```
Comment (gunasekaran-t) [inline roles/os10_interface/templates/os10_interface.j2:147]: If possible please remove the whilespace changes.

**Comment by cpu-lz** [comment]
added "no speed" and the old spaces.

**Comment by gunasekaran-t** [comment]
@cpu-lz Thanks for your contribution.

**Pull request merged**